buildSrc/src/main/groovy/io.deephaven.java-toolchain-conventions.gradle

```diff
@@ -64,15 +64,12 @@ tasks.withType(JavaCompile).configureEach {
   options.compilerArgs << '-parameters'
 
   if (!excludeReleaseFlag) {
-    if (name == 'compileTestJava') {
-      if (compilerVersion != testLanguageLevel) {
+    def checkLevel = (name == 'compileTestJava') ? testLanguageLevel : languageLevel
+    if (compilerVersion != checkLevel) {
         options.release.set testLanguageLevel
-      }
-    } else {
-      if (compilerVersion != languageLevel) {
-        options.release.set languageLevel
-      }
     }
+  } else {
+    sourceCompatibility = languageLevel
```

Comment:
Let's make a comment here that this is a special case - I'm not sure why add-exports doesn't work w/o this (while the failure w/ --release makes a bit of sense).